consul-l7-observability-kubernetes/index.json

```
@@ -0,0 +1,74 @@
+{
+  "title": "Consul Service Mesh L7 Observability on Kubernetes with Prometheus and Grafana",
+  "description": "In this hands-on lab you will install Consul service mesh, Prometheus, and Grafana on Kubernetes.",
+  "difficulty": "Intermediate",
+  "time": "15 minutes",
+  "details": {
+    "steps": [
+      {
+        "title": "Provisioning Lab Infrastructure",
+        "text": "provision.md",
+        "courseData": "provision.courseData.sh",
+        "code": "provision.code.sh",
+        "verify": "provision.verify.sh"
+      },
+      {
+        "title": "Install Consul service mesh on Kubernetes",
+        "text": "step1.md"
+      },
+      {
+        "title": "Install Prometheus",
+        "text": "step2.md"
+      },
+      {
+        "title": "Install Grafana",
+        "text": "step3.md"
+      },
+      {
+        "title": "Install the demo application",
+        "text": "step4.md"
+      },
+      {
+        "title": "Generate traffic",
+        "text": "step5.md"
+      },
+      {
+        "title": "View the traffic in Grafana",
+        "text": "step6.md"
+      },
+      {
+        "title": "View the service mesh in Consul",
+        "text": "step7.md"
+      }
+    ],
+    "intro": {
+      "text": "intro.md"
```

```suggestion
      "text": "intro.md",
       "credits": "https://learn.hashicorp.com/consul"
```

Add a link to Learn in the footer of intro and credits.